samples/modules/thrift/hello_server/Kconfig

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2022 Meta
+
+source "Kconfig.zephyr"
+
+config THRIFT_COMPACT_PROTOCOL
+  bool "Enable TCompactProtocol in samples"
+  depends on THRIFT
+  help
+    Enable this option to use  TCompactProtocol in samples
```

Comment:
as prior - also note that all Kconfigs use 1x tab, for the help section it is 1x tab followed by 2x spaces